Update CONTRIBUTING.md to include notes on closure of bugs (#305)
microsoft/ebpf-for-windows: CONTRIBUTING.md

If you found a non-security related bug, you can help us by
[submitting a GitHub Issue](https://github.com/microsoft/ebpf-for-windows/issues/new).
The best bug reports provide a detailed description of the issue and step-by-step instructions
for reliably reproducing the issue. 

We will aim to triage issues in weekly triage meetings. In case we are unable to repro the issue, we will request more information from you, the filer.

> There will be a waiting period of 2 weeks for the requested information and if there is no response, the issue will be closed. If this happens, please consider reopening the issue if do get to collecting the requested information. 

Reviewer: ```suggestion
There will be a waiting period of 2 weeks for the requested information and if there is no response, the issue will be closed. If this happens, please reopen the issue if you do get a repro and collect the requested information. 
```
Fix grammar ("if do...")

Author: thanks for catching it.
